src/main/java/com/github/arkadiusz97/sms_filter_service/smsfilterservice/sms_operator_service/repository/ClientRepository.java

```diff
@@ -0,0 +1,11 @@
+package com.github.arkadiusz97.sms_filter_service.smsfilterservice.sms_operator_service.repository;
+
+import com.github.arkadiusz97.sms_filter_service.smsfilterservice.sms_operator_service.domain.Client;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.List;
+
+public interface ClientRepository extends JpaRepository<Client, Integer> {
+    List<Client> findByNumber(String number);

```

Comment:
Czy używamy gdzieś `findByNumber` ?

Reply:
To zbędna metoda, którą powinno się usunąć. Użyłem jej raz na samym początku, ale potem zastąpiłem ją `findOneByNumber`.